### DISCUSSION TITLE
[BUG] No puedo eliminar tasks desde canal en forma aleatoria

### DISCUSSION THREAD

**Discussion by seiturra** [Cotalker/documentation#99] [4. Bug Reports] [answered]
Buenas a todos,

Detecté un bug algo difícil de replicar. Sólo me ha pasado una vez pero puede ser la causa de un ticket que tuvimos hace una semana.

El bug es que a veces no se puede eliminar una task haciendo click en el botón del panel lateral para desactivar las task. Cuando me pasó presioné más de una vez el botón en un lapso de aprox. 10 segundos. Esto ocurrió ayer 19/04 a las 19:51~19:53.

Tenía svc-scheduler-process abierto y no sé si es una coincidencia o no pero justo en ese momento decía el error que aparece frecuentemente queue is full

_APM Server transport error (503): Unexpected APM Server response
APM Server accepted 0 events in the last request
Error: queue is full_

Saludos.

**Comment by acedward** [comment]
En particular:
```
APM Server transport error (503): Unexpected APM Server response
APM Server accepted 0 events in the last request
Error: queue is full
```
Es un error del logger interno, no afecta al proceso en lo absoluto.

Que una task no pueda ser eliminada es más problemático. Tienes más información

**Reply by seiturra** [reply]
sólo lo que escribí

**Reply by seiturra** [reply]
Dejo explícito que luego de esos ~10 segundos sí pude eliminar la tarea.

**Comment by seiturra** [comment]
@acedward 
Edit: Este bug no está solved, lo que estoy reportando no es que no se pueda eliminar la task, es que no se pueden eliminar tasks en ciertos momentos. 
Me acaba de suceder nuevamente.
Este problema podría traernos más tickets como ya sucedió anteriormente. Ante tal caso ¿Cómo respondemos?

**Reply by seiturra** [reply]
https://drive.google.com/file/d/1-WV_ioeIV_DTBo4Khjmi2k4RxXEhANbR/view?usp=sharing

**Comment by acedward** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/3951895/115449725-19f6f900-a1e9-11eb-84f2-c89ea749f1d8.png)

Pude reproducir, esto aparece en network cuando pasa.

**Reply by acedward** [reply]
@guillevalin @rafaelwalter

**Reply by guillevalin** [reply]
Se creó ticket interno para ser resuelto como bug de categoría media.

**Reply by acedward** [reply]
No hay más feedback. Voy a cerrar.
